src/DynamoCore/Library/FunctionDescriptor.cs

```diff
@@ -170,7 +170,7 @@ public string Summary
         /// </summary>
         public string Description
         {
-            get { return !String.IsNullOrEmpty(Summary) ? Summary + "\n\n" + Signature : Signature; }
+            get { return !String.IsNullOrEmpty(Summary) ? Summary : string.Empty; }
```

Comment:
I would prefer `get { return String.IsNullOrEmpty(Summary) ? string.Empty : Summary; }`, but this is okay too.